core/src/start.rs

```diff
@@ -122,6 +125,19 @@ pub async fn start_rindexer(details: StartDetails<'_>) -> Result<(), StartRindex
 
             let manifest = Arc::new(read_manifest(details.manifest_path)?);
 
+            // Channels for Reth block data
+            let mut reth_channels = RethChannels::new();
+
+            // Start Reth nodes for enabled networks
+            for network in manifest.reth_enabled_networks() {
+                let reth_cli = network.reth.as_ref().unwrap().cli_config.to_reth_cli();
+                info!("Starting Reth node for network: {}", network.name);
+                let reth_tx = start_reth_node_with_exex(reth_cli)?;
+                info!("Started Reth node for network: {}", network.name);
```

Comment:
will reth native node logs come at same time as rindexer? could you see that being a pain or easy to see both mixed?

Reply:
It didnt hurt dev experience much during my testing, since it is pretty useful to track exex logs inside of reth as well. But the user could configure reth to write logs to a file instead (using cli args) and avoid seeing them.